packages/components-native/src/utils/meta/meta.json

```diff
@@ -0,0 +1,68 @@
+{
+  "exportedComponents": [
+    "ActionItem",
+    "ActionItemGroup",
+    "ActionLabel",
+    "ActivityIndicator",
+    "AtlantisContext.Provider",
+    "AtlantisContext.Consumer",
```

Comment:
part of me finds these Consumer entries kind of weird

on one hand, I don't expect anyone to ever use that since it's been replaced by `useContext`

on the other, it is correct that it is something we are exporting when we do `export SomeContext = React.createContext("whatever");`